changelog.d/6042.feature

```diff
@@ -0,0 +1 @@
+Allow homeserver to handle threepid validation itself when adding a threepid to a user's account.
```

Comment:
this only happens for emails: s/threepid/email address/?

Reply:
The homeserver "handles" msisdns by passing the work off to a trusted identity server transparent to the user.

Reply:
Have changed to "handle or delegate".

Reply:
I don't follow. We're not changing anything to do with msisdns in this change. Why are they relevant?

Reply:
I think I was getting confused with the msisdn stuff in this PR. I've switched the changelog entry to just mention emails now.